src/torchio/transforms/preprocessing/spatial/to_orientation.py

```diff
@@ -0,0 +1,97 @@
+import nibabel as nib
+import numpy as np
+import torch
+
+from ....data.subject import Subject
+from ...spatial_transform import SpatialTransform
+
+
+class ToOrientation(SpatialTransform):
+    """Reorient the data to a specified orientation.
+
+    This transform reorders the voxels and modifies the affine matrix to match
+    the specified orientation code.
+
+    Common orientation codes include:
+
+    - "RAS": (neurological convention)
+        - The first axis goes from Left to Right (R).
+        - The second axis goes from Posterior to Anterior (A).
+        - The third axis goes from Inferior to Superior (S).
+    - "LAS": (radiological convention)
+        - The first axis goes from Right to Left (L).
+        - The second axis goes from Posterior to Anterior (A).
+        - The third axis goes from Inferior to Superior (S).
+
+    See `NiBabel docs about image orientation`_ for more information.
+
+    Args:
+        orientation: A three-letter orientation code.
+        **kwargs: See :class:`~torchio.transforms.Transform` for additional
+            keyword arguments.
+
+    .. _NiBabel docs about image orientation: https://nipy.org/nibabel/image_orientation.html
+    """
+
+    def __init__(self, orientation='RAS', **kwargs):
+        super().__init__(**kwargs)
+        if not isinstance(orientation, str) or len(orientation) != 3:
+            raise ValueError(
+                f'Orientation must be a 3-letter string, got {orientation}'
+            )
+
+        valid_codes = set('RLAPIS')
+        if not all(c in valid_codes for c in orientation):
```

Comment:
can you also add a check that none of them is duplicated?
```suggestion
        if not all(c in valid_codes for c in orientation) or len(set(orientation)) != len(orientation):
```

Reply:
Wouldn't that be covered by the rest of the checks? I think the case is covered in the unit tests.

Reply:
it's not covered. The tests check for it to be of length 3, but not for duplicated values (e.g. RAR or LAL)

Reply:
you're right. It checks the axes separately, my bad :)